package.json

```diff
@@ -17,6 +17,7 @@
     "generate-clients": "yarn generate-client-paas-mpa"
   },
   "dependencies": {
+    "@aws-amplify/auth": "^5.6.6",
```

Comment:
Do we need to specify this separately? Is the version in sync with `aws-amplify`?